libs/eavmlib/src/port.erl

```diff
@@ -50,7 +50,7 @@
 %% return a term or `{error, Reason}'.
 %% @end
 %%-----------------------------------------------------------------------------
--spec call(pid(), Message :: term()) -> term().
+-spec call(Pid :: pid(), Message :: term()) -> term().
```

Comment:
We may want to call this parameter `Port` to match the code, or update code to call it `Pid`.